### ISSUE TITLE
[Bug]: Plugin Error

### ISSUE THREAD

**Issue by sahandilshan** [ballerina-platform/ballerina-lang#43245]
### Description

Getting the following error after installing the latest Ballerina and the latest VSCode Ballerina extension

<img width="484" alt="Screenshot 2024-08-06 at 1 55 50 PM" src="https://github.com/user-attachments/assets/5a098a59-7436-44e2-8e78-da45a2dce256">
 
Error log
```
Error executing 'bal home'.
<---- cmd output ---->
Error: LinkageError occurred while loading main class io.ballerina.cli.launcher.Main
	java.lang.UnsupportedClassVersionError: io/ballerina/cli/launcher/Main has been compiled by a more recent version of the Java Runtime (class file version 61.0), this version of the Java Runtime only recognizes class file versions up to 55.0
<---- cmd output ---->
Plugin version: 4.7.4
Ballerina version: Update Tool 1.4.2
Failed to activate Ballerina extension. Cannot read properties of undefined (reading 'onNotification')
```

### Steps to Reproduce

1. Set the jave home to 11 or lesser version
2. Install Ballerina from brew
3. Download the VSCode extension and you will get the above error message

### Affected Version(s)

v2201.9.2

### Related area

-> Compilation

**Comment by SoumyaEXE** [comment]
hello @xlight05 ,
could you please assign this to me also. I want to give a try to this issue once also.

**Comment by divyeshtechs-prog** [comment]
The error occurs because the installed version of Ballerina was compiled using Java 17, while your system is currently running Java 11. Java 11 cannot run programs compiled with newer Java versions. To fix this issue, you have two options:
Option 1: Use an older version of Ballerina that supports Java 11. Uninstall your current Ballerina version and download an older release, such as Swan Lake Beta 3, from the official Ballerina GitHub releases page. Extract it and install it manually. This allows you to continue using Java 11.
Option 2: Install Java 17 but keep Java 11 as your system default. You can install OpenJDK 17 and temporarily set it as the environment for Ballerina by running:
export JAVA_HOME=/usr/lib/jvm/java-17-openjdk-amd64
export PATH=$JAVA_HOME/bin:$PATH
Then run Ballerina commands such as bal home. This way, Ballerina will use Java 17 only for that session, without changing your system’s default Java version.
In summary, the problem happens because of a Java version mismatch. You can either downgrade Ballerina to match Java 11 or temporarily use Java 17 only for Ballerina to resolve the error.

**Comment by mannubaveja007** [comment]
Can I work on this issue?
